scripts/provisioning/setupcmd.py

```diff
@@ -118,14 +118,21 @@ def get_confvalue(self, key: str):
     assert self.provisioner_confstore != None
     return self.provisioner_confstore.get_config(key)
 
+  def get_confvalue_with_validation(self, key: str):
+    assert self.provisioner_confstore != None
+    value = self.provisioner_confstore.get_config(self.get_confkey(key))
+    if value is None:
+      value = self.s3_confkeys_store.get_config("DEFAULT_"+ key)
```

Comment:
Use try construct and check for None value
1)if exception happens from lower layer or if value is None, throw exception here itself.